{{cookiecutter.project_name}}/.pylintrc.toml

```diff
@@ -0,0 +1,52 @@
+[tool.pylint.main]
+ignore = [
+    ".git",
+    "__pycache__",
+    ".venv",
+    "venv",
+    "tests",
+    "migrations",
+    "settings",
+    "snapshots",
+]
+disable = [
+    "missing-module-docstring",
+    "missing-class-docstring",
+    "missing-function-docstring",
+    "unused-argument",
+    "redefined-builtin",
+    "import-outside-toplevel",
+    "no-else-return",
+    "unspecified-encoding",
+    "duplicate-code",
+    "super-with-arguments",
```

Comment:
Is it useful to ignore?

Reply:
Hm, I guess no - it would be better to get an error and take care of ignoring this rule if needed